workloads/router-perf-v2/common.sh

```diff
@@ -0,0 +1,111 @@
+ENGINE=${ENGINE:-podman}
+INFRA_TEMPLATE=http-perf.yml.tmpl
+INFRA_CONFIG=http-perf.yml
+KUBE_BURNER_IMAGE=quay.io/cloud-bulldozer/kube-burner:latest
+URL_PATH=${URL_PATH:-"/1024.html"}
+TERMINATIONS=${TERMINATIONS:-"http edge passthrough reencrypt mix"}
+KEEPALIVE_REQUESTS=${KEEPALIVE_REQUESTS:-"1 10 100"}
+CLIENTS=${CLIENTS:-"1 40 200"}
+SAMPLES=${SAMPLES:-1}
+QUIET_PERIOD=${QUIET_PERIOD:-10s}
+NUMBER_OF_ROUTERS=${NUMBER_OF_ROUTERS:-2}
+
+export TLS_REUSE=${TLS_REUSE:-true}
+export UUID=$(uuidgen)
+export RUNTIME=${RUNTIME:-120}
+export ES_SERVER=${ES_SERVER:-https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443}
+export ES_INDEX=${ES_INDEX:-router-test-results}
+export RAMP_UP=${RAMP_UP:-0}
+export HOST_NETWORK=${HOST_NETWORK:-true}
+export KUBECONFIG=${KUBECONFIG:-~/.kube/config}
+export NODE_SELECTOR=${NODE_SELECTOR:-'{"node-role.kubernetes.io/workload": ""}'}
+export NUMBER_OF_ROUTES=${NUMBER_OF_ROUTES:-100}
+export CERBERUS_URL=${CERBERUS_URL}
+
+log(){
+  echo -e "\033[1m$(date "+%d-%m-%YT%H:%M:%S") ${@}\033[0m"
+}
+
+deploy_infra(){
+  log "Deploying benchmark infrastructure"
+  envsubst < ${INFRA_TEMPLATE} > ${INFRA_CONFIG}
+  ${ENGINE} run --rm -v $(pwd)/templates:/templates -v ${KUBECONFIG}:/root/.kube/config -v $(pwd)/${INFRA_CONFIG}:/http-perf.yml ${KUBE_BURNER_IMAGE} init -c http-perf.yml --uuid=${UUID}
+  kubectl create configmap -n http-scale-client workload --from-file=workload.py
+  log "Adding workload.py to the client pod"
+  oc set volumes -n http-scale-client deploy/http-scale-client --type=configmap --mount-path=/workload --configmap-name=workload --add
+  kubectl rollout status -n http-scale-client deploy/http-scale-client
+  client_pod=$(kubectl get pod -l app=http-scale-client -n http-scale-client | grep Running | awk '{print $1}')
+}
+
+tune_liveness_probe(){
+  log "Disabling cluster version and ingress operators"
+  kubectl scale --replicas=0 -n openshift-cluster-version deploy/cluster-version-operator
```

Comment:
Maybe use one of the clients - either kubectl or oc for the operations to be consistent? Thoughts?

Reply:
yeah, makes sense, changing it.